jobserv/api/worker.py

```diff
@@ -55,7 +55,6 @@ def wrapper(*args, **kwargs):
                 worker = Worker(w.name, "?", 1, 1, "?", "", 1, w.allowed_tags)
                 worker.enlisted = True
                 db.session.add(worker)
```

Comment:
If a worker "registers" itself then why do we need to call `db.session.add(worker)` at all?

Reply:
scale workers using JWT "register" by calling the update worker API. `db.session.add` is the way you tell the ORM to create a new entry in the DB (once db.session.commit is called)

Reply:
its a bit like the device-gateway where we create some devices on the fly